env_lcg.sh

```
@@ -1,8 +1,13 @@
-LCG_RELEASE=dev3 # or dev3 for ROOT master (dev4 is the latest ROOT stable branch)
+LCG_RELEASE=dev3 # dev3 for ROOT master (dev4 is the latest ROOT stable branch)
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)